src/com/company/M10/Task2.java

```diff
@@ -0,0 +1,49 @@
+package com.company.M10;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Task2 {
+
+    public static final String SPACE = " ";
+    public static final int NAME_PLACE = 0;
+    public static final int AGE_PLACE = 1;
+    public static final int TWO_VALUES = 2;
```

Comment:
let's give move accurate names for the constants that explain business logic, e.g.:
```suggestion
    public static final String DELIMITER = " ";
    public static final int NAME_INDEX = 0;
    public static final int AGE_INDEX = 1;
    public static final int COLUMNS_COUNT = 2;
```